### PR TITLE
Add user email in navbar

### PR DESCRIPTION
**Pull request by bluepostit**
Allows user to easily see their email address

### CODE REVIEW

File: app/views/shared/_navbar.html.erb [hunk 1]
```diff
@@ -18,6 +18,7 @@
           <%= link_to "Messages", "#", class: "nav-link" %>
         </li>
         <li class="nav-item dropdown">
+          <%= current_user.email %>
```
Comment (bluepostit) [inline app/views/shared/_navbar.html.erb:21]: Is this right?

Comment (mGor18) [reply app/views/shared/_navbar.html.erb:21]: Yes but with a different color.

File: app/views/shared/_navbar.html.erb [hunk 2]
```diff
@@ -18,6 +18,7 @@
           <%= link_to "Messages", "#", class: "nav-link" %>
         </li>
```
Comment (bluepostit) [inline app/views/shared/_navbar.html.erb:19]: This looks odd

File: app/views/shared/_navbar.html.erb [hunk 3]
```diff
@@ -18,6 +18,7 @@
...
         <li class="nav-item dropdown">
+          <%= current_user.email %>
           <%= image_tag "https://kitt.lewagon.com/placeholder/users/ssaunier", class: "avatar dropdown-toggle", id: "navbarDropdown", data: { toggle: "dropdown" }, 'aria-haspopup': true, 'aria-expanded': false %>
           <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown">
             <%= link_to "Action", "#", class: "dropdown-item" %>
```
Comment (bluepostit) [inline app/views/shared/_navbar.html.erb:24]: wonder about this

**Pull request merged**